src/components/input/input.stories.mdx

```diff
@@ -33,3 +41,64 @@ You can also add `c-input` to `<select>` elements, in which case you’ll get a
     })}
   </Story>
 </Preview>
+
+## Elastic textarea
+
+Add `js-elastic-textarea` to a `<textarea>` element\* to have it expand as the user types more than will fit.
+
+If the `rows` attribute is set, it will be used as the minimum height.
+
+<p>
+  <small>* see also the JavaScript usage notes below</small>
+</p>
+
+<Preview>
+  <Story name="Elastic Textarea" height="350px">
+    {() => {
+      // Use storybook hooks to trigger JS after story renders
+      // @see https://github.com/storybookjs/storybook/issues/7786
+      useEffect(() => {
+        // Select all elastic textarea elements, run create function on each
+        [...document.querySelectorAll('.js-elastic-textarea')].map(
```

Comment:
I'm not sure `map` is necessary if we're not storing the result of the `map`. Should this just be a `forEach`?

Reply:
Actually, is the `map`/`forEach` necessary if the demo only has one textarea? Could we just target a single node and simplify the code?

```js
useEffect(() => { 
   createElasticTextArea(document.querySelector('js-elastic-textarea');
}, []);
```

Reply:
Yes, for our Story, we can do that (and I will). However, I'll leave the code sample alone, because I think most people will want to target all elements with that class, not just the first one found.